app/src/main/java/org/eu/exodus_privacy/exodusprivacy/utils/CollectionUtils.kt

```diff
@@ -0,0 +1,5 @@
+package org.eu.exodus_privacy.exodusprivacy.utils
+
+inline fun <T> List<T>.updateAndGet(block: MutableList<T>.() -> Unit): List<T> {
```

Comment:
I dont think it is showing that warning for this line of code. It might be showing for `Throwable.propagateCancellation()`.

And the warning is - `Expected performance impact from inlining is insignificant.`

But inlining is necessary in this function to properly propagate cancellation.

Reply:
ok